real_estate/models/estate_property.py

```diff
@@ -0,0 +1,33 @@
+from odoo import fields, models
+
+class EstateProperty(models.Model):
```

Comment:
```suggestion


class EstateProperty(models.Model):
```
There should be two empty line before defining a class